internal/util/debug/debug.go

```diff
@@ -74,9 +74,21 @@ func RunHandler(ctx context.Context, addr string, r prometheus.Registerer, l *za
 
 	must.NoError(r.Register(requestCount))
 
+	// started handler, which is used for startup probe, returns StatusOK when FerretDB listener were initialized.
+	// If it wasn't yet, the StatusInternalServerError is returned.
```

Comment:
I think `503 Service Unavailable` is more suitable for this case https://www.rfc-editor.org/rfc/rfc7231#section-6.6.4.

Reply:
I wasn't sure about that, as it is usually used for the server overload (which is more suitable for the readiness probe)

Reply:
Hmm, I see. You are right about usage for server overload. So the current internal error is probably most appropriate :+1: